src/data.rs

```diff
@@ -17,6 +17,8 @@ pub struct Container {
     pub id:    String,
     pub name:  String,
     pub image: String,
+    pub namespace: String,
+    pub pod: String,
```

Comment:
Pod information will not be available when running outside k8s so perhaps these fields should go into a separate `Pod` structure stored in an `Option<Pod>` field of the `Process` struct?